tf-job-operator-chart/templates/crd.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
Why put CRD yaml in charts, I think charts is used for Apps not CRDs.

Reply:
Then where to place the CRD creation? I think we should create it in the charts, or the charts can not be run correctly.

Reply:
> Then where to place the CRD creation?

CRD creation is in the set-up stage.

Charts is excepted deployed be `helm` automatically.
If you want to create CRD automatically along with TFJobs, maybe we need add some dependency here (not just put):
- create CRD first
- deploy a TFJob later

Reply:
I will file an issue about the question.